packages/ketcher-react/src/script/ui/views/modal/components/Text/Text.jsx

```diff
@@ -0,0 +1,43 @@
+/****************************************************************************
```

Comment:
why not `.tsx`?

Reply:
fixed

Reply:
fixed

Reply:
Please, add type for props. See [TopToolbar.tsx](https://github.com/epam/ketcher/blob/master/packages/ketcher-react/src/script/ui/views/toolbars/TopToolbar/TopToolbar.tsx#L34) and [TopToolbar.container.ts](https://github.com/epam/ketcher/blob/master/packages/ketcher-react/src/script/ui/views/toolbars/TopToolbar/TopToolbar.container.ts#L23) for example